src/encoder/encoder.cpp

```diff
@@ -155,3 +155,29 @@ EncoderRecordingSettingsPointer EncoderFactory::getEncoderRecordingSettings(Enco
         return std::make_shared<EncoderWaveSettings>(pConfig, ENCODING_WAVE);
     }
 }
+
+void Encoder::addToTracklist(const QString& artist,
+        const QString& title,
+        std::chrono::seconds timecode) {
+    auto recordedDuration = timecode.count();
+    m_trackList.append(
+            QStringLiteral("%1: %2 - %3")
+                    .arg(QString("%1:%2:%3")
+                                    .arg(recordedDuration / (60 * 60),
+                                            2,
+                                            'f',
+                                            0,
+                                            '0') // hours
+                                    .arg((recordedDuration / 60) % 60,
+                                            2,
+                                            'f',
+                                            0,
+                                            '0') // minutes
+                                    .arg(recordedDuration % 60, 2, 'f', 0, '0'),
+                            artist.trimmed().isEmpty()
+                                    ? QObject::tr("(Unknown Artist)")
+                                    : artist,
+                            title.trimmed().isEmpty()
+                                    ? QObject::tr("(Unknown Title)")
+                                    : title));
```

Comment:
we already have logic for generating a tracklist in cuesheet format. IMO it would make more sense to simply write that in the comment.

Reply:
The [cuesheet](https://en.wikipedia.org/wiki/Cue_sheet_(computing)) have a specific, non-human friendly format. 

The point of this formatting is to generate something like
```
00:11:22: An artist - A title
```

Since the formatting logic is meant to be unrelated (a change to CUE shouldn't impact metadata and vice versa), I thought it was alright to have a little bit of code duplication (that is for the timecode only)